src/components/login/index.js

```diff
@@ -50,15 +50,16 @@ const LoginComponent = ({
                 <div className={`${styles.innerWrapper}`}>
                     <div className={styles.loginCode}>
                         {summitData?.secondary_logo && <img className="login-logo" src={`${summitData?.secondary_logo}`} />}
-                        {title}
+                        <div style={{ padding: "15px 0 0 0"}}>{title}</div>
                         <div className={styles.input}>
                             <input placeholder="youremail@example.com" value={email} onChange={e => setEmail(e.target.value)}
                                    onKeyPress={(ev) => ev.key === 'Enter' ? loginCode() : null} data-testid="email-input" />                           
                         </div>
                         <div onClick={() => loginCode()} data-testid="email-button"
                             className={`${styles.button} ${styles.email_login_button} ${email === '' ? `${styles.pointerDisabled} ${styles.buttonDisabled}` : `${styles.primaryEmailButton}`}`}>
-                            <EmailRoundedIcon style={{ fontSize: '20px' }} />
-                            <span>Email me a login code</span>
+                            <EmailRoundedIcon style={{ fontSize: "20px" }} />
```

Comment:
please use css classes , dont hardcode styles

Reply:
Size will not change on this SVG icon without this inline hardcode